src/components/common/sideMenu/DrawerBody.jsx

```diff
@@ -38,6 +46,24 @@ const DrawerBody = (props) => {
     navigate(LOGINPAGE);
   };
 
+  const handleUploadModalClose = () => {
+    setShowUploadModal(false);
+  };
+
+  const handleUploadDocument = (selectedFile) => {
+    // send doc to be
+    if (selectedFile != null) {
+      setSuccessMessage(SUCCES_MESSAGE);
+    }
+    setShowUploadModal(false);

```

Comment:
Isn't it the same as 
```suggestion
 const handleUploadDocument = (selectedFile) => setShowUploadModal(selectedFile? SUCCESS_MESSAGE  : false);

```